docs/user-guide/federation/federated-ingress.md

```diff
@@ -229,6 +229,38 @@ kept running, the Federated Ingress ensures that user traffic is
 automatically redirected away from the failed cluster to other
 available clusters.
 
+## Known issue
+
+GCE L7 load balancer backends and healthchecks are known to flap due
+to the firewall rules from the underlying clusters in federation
+clobbering each other. In order to workaround this problem, you can
+install the firewall rules manually to expose the targets of all the
+underlying clusters in your federation for each Federated Ingress
+object so that the health checks can pass and GCE L7 load balancer
+is stable. This can be done using the `gcloud` command line tool as follows:
+
+```shell
+gcloud compute firewall-rules create <firewall-rule-name> \
+  --source-ranges 130.211.0.0/22 --allow [<service-nodeports>] \
+  --target-tags [<target-tags>] \
+  --network <network-name>
+```
+
+where:
+1. `firewall-rule-name` can be any name.
```

Comment:
I think it is implicit. `gcloud/Cloud console` will complain otherwise. Saying it should be unique here will add more confusion than clarity here.